README.md

```diff
@@ -0,0 +1,180 @@
+# Django-GDPR [![Build Status](https://travis-ci.org/BrnoPCmaniak/django-GDPR.svg?branch=develop)](https://travis-ci.org/BrnoPCmaniak/django-GDPR)
+
+This libarary enables you to store user's consent for data retention easily
+and to anonymize/deanonymize user's data accordingly.
+
+For brief overview you can check example app in `tests` directory.
+
+# Quickstart
+
+Install django-gdpr with pip:
+
+```bash
+pip install django-gdpr
+```
+
+Add gdpr to your `INSTALLED_APPS`:
+
+```python
+INSTALLED_APPS = [
+    # Django apps...
+    'gdpr',
+]
+```
+
+Imagine having a customer model:
+
+```python
+# app/models.py
+
+from django.db import models
+
+from gdpr.mixins import AnonymizationModel
+
+class Customer(AnonymizationModel):
+    # Keys for pseudoanonymization
```

Comment:
z té dokumentace jsem moc nepochopil jak ta pseudoanonymizace funguje. Kde se nastavuje, které pole to jsou?

Reply:
Každopádně ten komentář tady trochu mate, jednak to jsou fields a druhak tady se to imho nenastavuje, které fieldy se budou pseudoanonymizovat.

Reply:
to tam možná ještě nebylo, když jsem to psal